mlbench_core/evaluation/__init__.py

```diff
@@ -1 +1,32 @@
-from . import pytorch
\ No newline at end of file
+class AverageMeter(object):
```

Comment:
__init__.py should only be used to setup a package. I.e. exposing key classes/functionality from the package.

It shouldn't contain the code itself, to keep things clean and separated.